libs/src/services/ABIDecoder/index.ts

```diff
@@ -18,8 +20,9 @@ loadABI('PlaylistFactory.json')
 loadABI('UserLibraryFactory.json')
 loadABI('UserReplicaSetManager.json')
 
-class AudiusABIDecoder {
-  static decodeMethod (contractName, encodedABI) {
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class -- should just use esm
```

Comment:
what's the todo here? just export a const?

Reply:
yeah eslint is saying that a class with only static methods should just be an object or just do export esm style for all the methods